components/Header/styles.module.scss

```diff
@@ -24,10 +24,19 @@
         margin-right: 1.25rem;
       }
 
-      button {
+      .button {
```

Comment:
Dúvida de quem há anos já não toca em CSS :joy: 

Fazer isso não diminui a semântica e possivelmente impacta em leitores de tela?

Reply:
Na verdade eu não fiz isso por conta do css em si, mas pelo fato de eu ter validado o site no [W3C Validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fbrasilapi.com.br%2F) e ele ter acusado como um problema 
![image](https://user-images.githubusercontent.com/16273730/153665802-68b34113-dff9-490c-a99f-37d6b41a2865.png)

Pelo que entendi um botão não deve ficar dentro de um `<a>` https://stackoverflow.com/a/6393863/9719107

Reply:
Boa noite, caros!
Estou meio sumido, mas estou vivo ainda... rs
Bom, com relação a semântica temos duas questões que avaliar =P

1. Onde buttons devem ser usados?
2. Onde links devem ser usados?

Tags <button/> são desenvolvidas pensando na executar ações, (submit e reset de forms ou customizadas usando javascript) sendo assim elas teoricamente, elas raramente apareceriam fora de uma tag </form>, se a única razão de existir do botão é ser um link, então a tag correta seria o <a>, e para os leitores de telas teríamos o próprio texto do botão e a prop title.

Para dar uma olhadela: https://stackoverflow.com/questions/64443645/html-semantics-button-acting-as-an-anchor

Agora se quisermos dar outro passo em direção à acessibilidade, estamos falando de propriedades aria:
[https://developer.mozilla.org/pt-BR/docs/Web/Accessibility/ARIA](https://developer.mozilla.org/pt-BR/docs/Web/Accessibility/ARIA)

E ai fugimos do validador da w3c, que valida basicamente a estrutura do html e css e passamos para validadores de a11y, como este:
[https://wave.webaim.org/](https://wave.webaim.org/)

Espero que meus 2 centavos ajudem em algo.. rs

Reply:
Eita, agora que eu vi que o github bagunçou tudo minhas tags na resposta (me culpa, claro!).
Mas sim, o que temos é um link com crise de identidade, ele tem a aparência de um botão, mas o comportamento de um link, neste caso, eu usaria `<a/>`

Reply:
Mas é ruim ter um link com aparência de botão?

Reply:
Já vi o Next e a Vercel colocando links assim

Reply:
Simbora então galera, to de acordo! ✅ 

(Era só dúvida msm pra aprender 😁 )

Reply:
> Na minha humilde opinião, nem um pouco. Acho que é mais uma decisão de design (visual, não de projetos... rs) e gosto do que algo que seja um impeditivo no uso. O importante é o html estar semanticamente correto, realmente.

Tudo bem, é que é uma coisa que também não entendo muito

Reply:
Tranquilo, a ideia é que todos possamos dar sugestão para achar a melhor solução possível para aquele detalhe que foi apontado.